src/components/atoms/Widget.container.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from 'react';
+import { Card } from '@material-ui/core';
+
+interface IProps {
+
+}
+export const WidgetContainer: FunctionComponent<IProps> = ({ children }) => {
```

Comment:
Why the `.container` in the filename?
Components' filenames are written in `UpperCamelCase'.